src/main/java/bio/terra/workspace/service/datareference/flight/DataReferenceFlightMapKeys.java

```diff
@@ -4,7 +4,7 @@
   public static final String REFERENCE_ID = "referenceId";
   public static final String WORKSPACE_ID = "workspaceId";
   public static final String NAME = "name";
-  public static final String REFERENCE_DESCRIPTION = "referenceDescription";
+  public static final String DESCRIPTION = "WSM:description";
```

Comment:
nit: I'd leave this as "referenceDescription"; clarifying that this is "WSM" doesn't describe much. I don't feel very strongly about this.

Reply:
Yeah, this is all because we can't use `description` as a key, I'll add a clarifying comment.

Reply:
(I think the prefix with the colon makes it clearer that it's just namespace-ish)

Reply:
The root problem here is that we do not pass the DataReferenceRequest into the flight as the request, but (pretty needlessly) break it into little pieces.

As I understand it, we got here by:
* AutoValue is good - use that
* AutoValue class is a pain to use as a Stairway input parameter
* Dismantle class into its atoms

Anyway, not for this PR. I expect to change the usage of AutoValue classes one way or the other so we do not have to dismantle them to launch a flight.